### PR TITLE
Add tuna reverse min balance config

### PR DESCRIPTION
**Pull request by billfort**

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -23,6 +23,7 @@ type Config struct {
 	TunaGeoDBPath          string
 	TunaMeasureBandwidth   bool
 	TunaMeasureStoragePath string
+	TunaReverseMinBalance  string
```
Comment (yilunzhang) [inline config.go:26]: We can use `TunaMinBalance` because everything here is reverse

File: config.go [hunk 2]
```diff
@@ -45,6 +46,7 @@ var defaultConfig = Config{
 	TunaGeoDBPath:          "",
 	TunaMeasureBandwidth:   false,
 	TunaMeasureStoragePath: "",
+	TunaReverseMinBalance:  "0.0",
```
Comment (yilunzhang) [inline config.go:49]: "0" is enough

File: config.go (hunk 1 above)
Comment (billfort) [reply config.go:26]: Yes, modified

File: config.go (hunk 2 above)
Comment (billfort) [reply config.go:49]: Yes, modified.

**Review by yilunzhang — APPROVED** [review]
LGTM

**Pull request merged**